cert/cert.proto

```diff
@@ -278,6 +278,10 @@ message LoadCertificateRequest {
   // certificates should squash the existing bundle. This field provides a
   // simplified means to provision a CA bundle that can be used to validate
   // other peer's certificates.
+  // To improve performance in the Target, certificates can be ordered.
+  // Groups of chained certificates should be last, where within, the root
+  // certificate is the last one. E.g.:
+  // CertA, CertB, CertB-Root, CertC, CertC-Intermediate, CertC-Root
```

Comment:
Sam, looks good. Please note there could also be CA-Cert for target certificate in the bundle.

Reply:
Thanks Reshad.
Indeed, I have created another PR to cover that under the simplified security model document.